formats/sd-aarch64.nix

```
@@ -1,52 +1,56 @@
 { config, lib, pkgs, modulesPath, ... }:
-let
-  extlinux-conf-builder =
-    import "${toString modulesPath}/system/boot/loader/generic-extlinux-compatible/extlinux-conf-builder.nix" {
-      pkgs = pkgs.buildPackages;
-    };
-
-in {
+{
   imports = [
     "${toString modulesPath}/installer/sd-card/sd-image.nix"
```

If this would just work without us having to duplicate all the code below, than I am fine with that:

```suggestion
    "${toString modulesPath}/installer/sd-card/sd-image.nix"
    "${toString modulesPath}/installer/sd-card/sd-image-aarch64.nix"
```

Less code to maintain in nixos-generators.